server/coordinator/procedure/dispatch/dispatch.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package dispatch
+
+import (
+	"github.com/CeresDB/ceresdbproto/pkg/clusterpb"
+)
+
+type EventDispatch interface {
+	SendOpenEvent(ShardIDs []uint32, targetNode string) (bool, error)
+
+	SendCloseEvent(ShardIDs []uint32, targetNode string) (bool, error)
+
+	SendChangeRoleEvent(ShardID uint32, shardRole clusterpb.ShardRole, targetNode string) (bool, error)
```

Comment:
The `send` and `event` in `SendOpenEvent` are verbose for `EventDispatch` interface, how about:
```suggestion
	OpenShards(ShardIDs []uint32, targetNode string) (bool, error)

	CloseShards(ShardIDs []uint32, targetNode string) (bool, error)

	ChangeShardRole(ShardID uint32, shardRole clusterpb.ShardRole, targetNode string) (bool, error)
```